.github/workflows/realm-generator.yml

```diff
@@ -0,0 +1,64 @@
+name: Realm Generator
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+  pull_request:
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+jobs:
+  Generator_CI:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest] # windows is not tested ue to bug in how error messages are rendered
+        channel: [stable, beta]
+
+    defaults:
+      run:
+        working-directory: ./generator/
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name : Setup Dart SDK
+        uses: dart-lang/setup-dart@main
+        with:
+          sdk: ${{ matrix.channel }}
+
+      - name: Install dependencies
+        run: dart pub get
+
+      - name: Run generator tests
+        run: dart test --reporter expanded --coverage ./coverage/
+
+      - name: Generate generator coverage report
+        run: |
+          dart run coverage:format_coverage \
+            --in coverage/ \
+            --out ./coverage/lcov.info \
+            --check-ignore \
+            --lcov \
+            --packages .packages \
+            --report-on lib
+
+      - name: Publish Generator Coverage
+        id: publish-generator-coverage
+        uses: coverallsapp/github-action@1.1.3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          flag-name: realm_generator
+          path-to-lcov: ./coverage/lcov.info
+        continue-on-error: true
+
+      - name: Output Coveralls response
+        run: echo ${{ steps.publish-generator-coverage.outputs.coveralls-api-result }}
```

Comment:
If the above fails does this fail and should it be skipped if the above fails?

Reply:
No, but let me test for `${{ success() }}`

Reply:
ok I think my question is not valid since this has if {success}